__tests__/app.test.js

```diff
@@ -232,3 +232,59 @@ describe("GET /api/articles/:article_id/comments", () => {
 
 })
 
+
+
+describe("GET /api/articles?topic=(any) ", () => {
+  test("200 sends an array of articles with a topic of mitch to the client", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles.length).toBe(12);
+        body.articles.forEach((article) => {
+          expect(article.topic).toBe("mitch");
+          expect(typeof article.title).toBe("string");
+          expect(typeof article.author).toBe("string");
+          expect(typeof article.created_at).toBe("string");
+          expect(typeof article.article_id).toBe("number");
+          expect(typeof article.article_img_url).toBe("string");
+          expect(typeof article.comment_count).toBe("string");
```

Comment:
Could use `toMatchObject` here as well if you wanted to do a bit less typing
This works fine though! 👍